addons/mail_bot/static/src/js/notification_alert.js

```diff
@@ -8,15 +8,49 @@ var widgetRegistry = require('web.widget_registry');
 // Display Notification alert on user preferences form view
 // -----------------------------------------------------------------------------
 var NotificationAlert = Widget.extend({
-   template: 'mail.NotificationAlert',
-   /**
-    * @override
-    */
-   init: function () {
-      this._super.apply(this, arguments);
-      var hasRequest = this.call('mailbot_service', 'isRequestingForNativeNotifications');
-      this.isNotificationBlocked = window.Notification && window.Notification.permission !== "granted" && !hasRequest;
-   },
+    template: 'mail.NotificationAlert',
+    /**
+     * @override
+     */
+    init() {
+        this._super(...arguments);
+        const hasRequest = this.call('mailbot_service', 'isRequestingForNativeNotifications');
+        this.isNotificationBlocked = window.Notification && window.Notification.permission !== "granted" && !hasRequest;
+        if (this.isNotificationBlocked) {
+            if (navigator.userAgent.includes("Chrome") == true) {
+                this.settingUrl =  "chrome://settings/content/siteDetails?site=https%3A%2F%2F" + window.location.host;
+            } else if (navigator.userAgent.includes("Firefox") == true) {
+                this.settingUrl = "about:preferences#privacy";
+            } else if (navigator.userAgent.includes("Safari") == true) {
+                this.settingPath = "(path:Safari > Preferences > Website > Notifications)";
+            }
+        }
+    },
+    /**
+     * Copies the browser settings URL link to the clipboard.
+     *
+     * @override
+     */
+    start() {
+        this._super(...arguments);
```

Comment:
`start()` is async function, so expect `async` keyword and `await` on `_super` call